Converters/Rhino/Speckle.Converters.RhinoShared/ToHost/Raw/RhinoPools.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Buffers;
+using Speckle.Sdk.Dependencies;
+
+namespace Speckle.Converters.Rhino.ToHost.Raw;
+
+public static class RhinoPools
```

Comment:
Could you add here why this pools are introduced etc? for performance reasons, what was the problem that it fixed